packages/vitest/src/node/pools/forks.ts

```diff
@@ -1,3 +1,4 @@
+import type { FileSpec } from '@vitest/runner/types/runner'
```

Comment:
`@vitest/runner/types/runner` doesn't exist when built. you need to expose it in `@vitest/runner`